public/styles.css

```diff
@@ -337,7 +334,7 @@ a {
     left: 0;
     top: 0;
     background-color: #333;
-    padding-top: 20px;
+    padding-top: 10vh;
```

Comment:
This is to give a bit of space between the image and the sidebar items. Works on all browsers that support view dimension based size handling.